prepare_env.ps1

```diff
@@ -3,34 +3,48 @@ param([string]$builddir='build')
 $devlibs_url = "https://github.com/H-uru/PlasmaPrefix/releases/download/2020.05.01/devlibs.zip"
 
 if (!(Test-Path -PathType Container $builddir)) {
-    Write-Host "Creating build folder at .\$builddir... " -noNewLine
+    Write-Host "Creating build folder at $builddir... " -noNewLine
     New-Item -ItemType directory $builddir | Out-Null
     Write-Host "OK" -foregroundColor Green
 }
 Set-Location $builddir
 $path = (Get-Location).Path
 
 if (!(Test-Path -PathType Container devlibs)) {
+    # Hide progress output
+    $ProgressPreference = 'SilentlyContinue'
+
+    $zippath = Join-Path -Path $path -ChildPath "devlibs.zip"
+    $libpath = Join-Path -Path $path -ChildPath "devlibs"
+
     Write-Host "Downloading development libraries... " -noNewLine
-    ## This only works in PS3+ so we'll download the file directly
-    # Invoke-WebRequest $devlibs_url -OutFile devlibs.zip
-    $client = New-Object System.Net.WebClient
-    $client.DownloadFile($devlibs_url, $path + "\devlibs.zip")
+    if ($PSVersionTable.PSVersion.Major -ge 3) {
+        Invoke-WebRequest $devlibs_url -OutFile $zippath
```

Comment:
PowerShell 3.0 was made available with Windows 8. Given that Windows 7 is no longer supported by Microsoft, I think we can safely dump the old code.